kubectl-fdb/cmd/remove.go

```diff
@@ -21,155 +21,33 @@
 package cmd
 
 import (
-	"fmt"
-	"log"
-
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 
-	"github.com/FoundationDB/fdb-kubernetes-operator/controllers"
-	corev1 "k8s.io/api/core/v1"
-
 	"github.com/spf13/cobra"
-	"k8s.io/apimachinery/pkg/runtime"
-	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	ctx "context"
-
-	fdbtypes "github.com/FoundationDB/fdb-kubernetes-operator/api/v1beta1"
 )
 
 func newRemoveCmd(streams genericclioptions.IOStreams, rootCmd *cobra.Command) *cobra.Command {
 	o := NewFDBOptions(streams)
 
 	cmd := &cobra.Command{
 		Use:   "remove",
-		Short: "Adds an instance (or multiple) to the remove list of the given cluster",
-		Long:  "Adds an instance (or multiple) to the remove list field of the given cluster",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			force, err := rootCmd.Flags().GetBool("force")
-			if err != nil {
-				return err
-			}
-			cluster, err := cmd.Flags().GetString("fdb-cluster")
-			if err != nil {
-				return err
-			}
-			instances, err := cmd.Flags().GetStringSlice("instances")
-			if err != nil {
-				return err
-			}
-			withExclusion, err := cmd.Flags().GetBool("exclusion")
-			if err != nil {
-				return err
-			}
-			withShrink, err := cmd.Flags().GetBool("shrink")
-			if err != nil {
-				return err
-			}
-
-			config, err := o.configFlags.ToRESTConfig()
-			if err != nil {
-				return err
-			}
-
-			scheme := runtime.NewScheme()
-			_ = clientgoscheme.AddToScheme(scheme)
-			_ = fdbtypes.AddToScheme(scheme)
-
-			kubeClient, err := client.New(config, client.Options{Scheme: scheme})
-			if err != nil {
-				return err
-			}
-
-			namespace, err := getNamespace(*o.configFlags.Namespace)
-			if err != nil {
-				return err
-			}
+		Short: "Subcommand to remove instances from a given cluster",
+		Long:  "Subcommand to remove instances from a given cluster",
 
-			return removeInstances(kubeClient, cluster, instances, namespace, withExclusion, withShrink, force)
+		RunE: func(c *cobra.Command, args []string) error {
+			return c.Help()
 		},
 		Example: `
 # Remove instances for a cluster in the current namespace
-kubectl fdb remove -c cluster -i instance-1 -i instance-2
+kubectl fdb remove instances -c cluster instance-1 instance-2
 
 # Remove instances for a cluster in the namespace default
-kubectl fdb -n default remove -c cluster -i instance-1 -i instance-2
+kubectl fdb -n default remove instances -c cluster instance-1 instance-2
```

Comment:
As I read the `instances` subcommand, this invocation would treat `instance-1` and `instance-2` as pod names. Is that correct? If so, should we update this example?

Reply:
Good catch!